packages-ts/starknet-gauntlet-starkgate/src/commands/L1-bridge/deploy.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
What about bridge initialization? The bridge will need to be initalized via a proxy, after it's deployed.

Have you seen this e2e test here: https://github.com/smartcontractkit/chainlink-starknet/blob/develop/packages-ts/integration-starkgate/test/tokenBridge.test.ts